doc/whats_new.rst

```diff
@@ -13,6 +13,8 @@ Changes
   orthogonality between temporal filter and confound removal, an additional
   temporal filter will be applied on the confounds before removing them. This
   is according to Lindquist et al. (2018).
+- image.clean_img now accepts a mask to restrict the cleaning of the image.
```

Comment:
small tip, if you could do something like this ":func:`nilearn.image.clean_img`", then it is useful for user to directly click on this and go directly to documentation . You can for instance, look below how a function or class is referred.